packages/lab/src/__tests__/__e2e__/progress/CircularProgress.cy.tsx

```diff
@@ -1,23 +1,22 @@
 import { CircularProgress } from "../../../progress";
 
 describe("GIVEN a CircularProgress", () => {
-  beforeEach(() => {
-    cy.mount(<CircularProgress value={50} />);
-  });
-
   it("SHOULD render progress circle", () => {
+    cy.mount(<CircularProgress />);
     cy.get('[data-testid="circular-progress"]').should("exist");
   });
 
-  it("SHOULD render progress circle with correct value", () => {
-    cy.get("[aria-valuenow=50]").should("exist");
-  });
-
-  it("SHOULD render progress circle with correct max value", () => {
+  it("SHOULD render progress circle with correct value and percentage", () => {
+    cy.mount(<CircularProgress value={50} />);
     cy.get("[aria-valuemax=100]").should("exist");
```

Comment:
None of these tests follow our best practices?

These should be more like:

```suggestion
    cy.findByRole("progressbar").should("have.attr", "aria-valuemax", "100");
```